include/deal.II/base/polynomial.h

```diff
@@ -585,6 +586,119 @@ namespace Polynomials
     static std::vector<Polynomial<double> >
     generate_complete_basis (const unsigned int p);
   };
+
+
+
+  /**
+   * Polynomials for a variant of Hermite polynomials with better condition
+   * number in the interpolation than the basis from
+   * HermiteInterpolation. This class is only implemented for degree at least
+   * three, $n\geq 3$.
+   *
+   * In analogy to the actual Hermite polynomials this basis evaluates the
+   * first polynomial $p_0$ to 1 in $x=0$ and has a both a zero value and zero
+   * derivative in $x=1$. Likewise, the last polynomial $p_n$ evaluates to 1
```

Comment:
In English, I think the proper wording would be "at $x=0$. There are a number of occurrences of this.